scripts/upgrade

```diff
@@ -15,36 +15,36 @@ source /usr/share/yunohost/helpers
 
 app=$YNH_APP_INSTANCE_NAME
 
-domain=$(ynh_app_setting_get $app domain)
-path_url=$(ynh_app_setting_get $app path)
-admin=$(ynh_app_setting_get $app admin)
-is_public=$(ynh_app_setting_get $app is_public)
-final_path=$(ynh_app_setting_get $app final_path)
-language=$(ynh_app_setting_get $app language)
+domain=$(ynh_app_setting_get "$app" domain)
+path_url=$(ynh_app_setting_get "$app" path)
+admin=$(ynh_app_setting_get "$app" admin)
+is_public=$(ynh_app_setting_get "$app" is_public)
+final_path=$(ynh_app_setting_get "$app" final_path)
+language=$(ynh_app_setting_get "$app" language)
 
 #=================================================
 # ENSURE DOWNWARD COMPATIBILITY
 #=================================================
 
 # Fix is_public as a boolean value
 if [ "$is_public" = "Yes" ]; then
-	ynh_app_setting_set $app is_public 1
+	ynh_app_setting_set "$app" is_public 1
 	is_public=1
 elif [ "$is_public" = "No" ]; then
-	ynh_app_setting_set $app is_public 0
+	ynh_app_setting_set "$app" is_public 0
```

Comment:
Some of the high quality apps don't quote like this in their install scripts.

See https://github.com/YunoHost-Apps/gitlab_ynh/blob/testing/scripts/install#L80.

I'm not sure but I would do what they do ...

Reply:
I have no opinion here. It looks like a quotation-mark-them-all frenzy, but with little side effects?

Reply:
https://www.shellcheck.net recommands it, and I never noticed any impact.